fluid/language_model/train.py

```diff
@@ -1,14 +1,18 @@
+import os
 import sys
 import time
 
 import numpy as np
 import math
 
 import paddle.fluid as fluid
-import paddle.v2 as paddle
+import paddle
 
 import utils
 
+# random seed must set before configuring the network.
+fluid.default_startup_program().random_seed = 102
```

Comment:
We'd better set the random seed configurable

Reply:
you mean  set like this？
SEED=102
fluid.default_startup_program().random_seed = SEED